pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "mouse-gui"
+version = "0.1.0"
+description = ""
+authors = ["wiktorwalczak <wikwalczak@student.agh.edu.pl>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "3.11.8"
+scikit-image = "^0.22.0"
+numpy = "^1.26.4"
+matplotlib = "^3.8.3"
+mouse = { path = "/Users/wiktorwalczak/Desktop/inzynierka/poetry/MoUSE", develop = true }
```

Comment:
shouldn't this be deconstructed into at least `f"{os.environ["HOME"]}/..." or better step above current directory (os.cwd()) something like `Path('.') / "../MoUSE"` since we expect those 2 directories to be in same directory (in your case `/Users/wiktorwalczak/Desktop/inzynierka/`)

Reply:
As we are public now maybe better idea would be to publish the package to pypi at some later stage ?
As for now I'd use link to GitHub repo as requirement, instead of link to local directory.

See https://python-poetry.org/docs/cli/#add

<img width="892" alt="Screenshot 2024-03-22 at 10 09 30" src="https://github.com/JosephTheMoUSE/MoUSE-GUI/assets/7374733/d726e227-475c-4d6d-95d4-e0c0d61c374d">